ethon.gemspec

```diff
@@ -18,7 +18,7 @@ Gem::Specification.new do |s|
   s.required_rubygems_version = ">= 1.3.6"
   s.license = 'MIT'
 
-  s.add_dependency('ffi', ['>= 1.3.0'])
+  s.add_dependency('ffi', ['>= 1.5.0'])
```

Comment:
Passing callbacks via varargs requires ffi 1.5.0

Reply:
Should this not be `1.15.0`?

Reply:
Oops, you're totally right. I mixed up 1.5 and 1.15.
Just fixed it.